Cross-compiled libraries get a suffix naming the system by default, but this
causes deviation from native-built libraries and should be disabled.

--- a/build/cmake/functions.cmake
+++ b/build/cmake/functions.cmake
@@ -219,9 +219,9 @@
     endif()
 
     set(cross_target)
-    if (CMAKE_CROSSCOMPILING)
-        set(cross_target "-${CMAKE_SYSTEM_NAME}")
-    endif()
+    #if (CMAKE_CROSSCOMPILING)
+    #    set(cross_target "-${CMAKE_SYSTEM_NAME}")
+    #endif()
 
     set(lib_prefix "lib")
     if(MSVC OR (WIN32 AND wxBUILD_SHARED))
